api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1081,6 +1081,8 @@ public static String getOpenmrsProperty(String property) {
 		
 		props
 				.add(new GlobalProperty(GP_DRUG_ORDER_DRUG_OTHER, "", "Specifies the uuid of the concept which represents drug other non coded"));
+		props.add(new GlobalProperty(LOGIN_URL, "",
```

Comment:
My comment did not mean removing the default value. Am just guiding you to something else. 😊

Reply:
kindly elaborate..

Reply:
do you mean removing the global declaration and combining it with the addition on props.add(new GlobalProperty (...... property here))

Reply:
First of all, get it back to what it was. Then show my original comment to your fellow apprentices and ask them what i could be driving at. 😊